shared/actions/search/index.js

```diff
@@ -187,9 +187,9 @@ function* search({payload: {term, service, searchKey}}: Constants.Search) {
   try {
     yield call(onIdlePromise, 1e3)
     const searchResults = yield call(_apiSearch, term, _serviceToApiServiceName(service))
-    const rows = searchResults.list.map((result: RawResult) => {
-      return _parseRawResultToRow(result, service || 'Keybase')
-    })
+    const rows = searchResults.list.map((result: RawResult) =>
```

Comment:
we originally had a plain object inside the map but this is a pattern we should avoid